po/es.po

```diff
@@ -1597,40 +1590,40 @@ msgstr "atributo no encontrado"
 
 #: src/tagsouppullparser.cpp:145
 msgid "EOF found while reading XML tag"
-msgstr "EOF encontrado al leer la etiqueta XML"
+msgstr "Fin de archivo (EOF) encontrado al leer una etiqueta XML"
 
 #: src/urlviewformaction.cpp:48 src/urlviewformaction.cpp:61
 msgid "No link selected!"
-msgstr "¡Ningún enlace seleccionado!"
+msgstr "Ningún enlace seleccionado."
```

Comment:
Ditto about punctuation.

Reply:
Withdrawn, [see explanation in a comment below](https://github.com/newsboat/newsboat/pull/874#issuecomment-612408710).